### PR TITLE
Add Guideline

### PR DESCRIPTION
**Pull request by alexander-dejeu**
* Add Guideline 401k Platform

### CODE REVIEW

**Comment by 40504** [comment]
I think this icon more suitable:
guideline

**Comment by alexander-dejeu** [comment]
Nice suggestion @40504! Will update to use this icon as it does not have any background

**Comment by phallobst** [comment]
Please rebase your branch to the current master.

**Comment by alexander-dejeu** [comment]
Hi @Carlgo11! Thank you for the tag - I think I resolved the merge conflicts earlier today, let me know if that is not the case.  Diff looks good from a merge conflict perspective.  Cheers!

**Comment by Carlgo11** [comment]
@alexander-dejeu if you look at the [commit list](https://github.com/2factorauth/twofactorauth/pull/5184/commits) you'll see there are a bunch of unrelated commits.

**Comment by alexander-dejeu** [comment]
Ophh! Good callout @Carlgo11 - let me take a look as to why that would be the case - thanks!

**Comment by Carlgo11** [comment]
Try
```PWSH
git add remote upstream https://github.com/2factorauth/twofactorauth.git
git fetch upstream

git reset --hard upstream/master
git cherry-pick e091c31503bb7e7525080f87958f4cf4f707a81e
git cherry-pick 8e8a1419478f1e85694ac5fc6d44fc15c9e602d9

git push origin alex/add-guideline -f
```

**Comment by alexander-dejeu** [comment]
You rock @Carlgo11! 🔥 Looks much more clean now :)

**Comment by alexander-dejeu** [comment]
Anything I can do to help move this from being inactive to active and then into master?

File: _data/finance.yml [hunk 1]
```diff
@@ -196,6 +196,13 @@ websites:
     facebook: gerberlife
     img: gerberlife.png
 
+  - name: Guideline
+    url: https://www.guideline.com
+    img: guideline.png
+    tfa:
+      - totp
+    doc: https://success.guideline.com/hc/articles/360034791412
+
```
Comment (Carlgo11) [inline _data/finance.yml:207]:
```suggestion
    regions:
      - us

```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
Sorry for the delay.

As SVGs are prefered, please use this logo instead:

**Review by Carlgo11 — APPROVED** [review]
Thanks! 👍

**Comment by alexander-dejeu** [comment]
Glad to see SVG is the preferred asset type! Thank you for pushing the PR along @Carlgo11!

**Review by phallobst — APPROVED** [review]

**Pull request merged**